session.go

```diff
@@ -418,11 +422,34 @@ func (s *session) WriteBytes(pkg []byte) (int, error) {
 		return 0, ErrSessionClosed
 	}
 
-	lg, err := s.Connection.send(pkg)
+	leftPackageSize, totalSize, writeSize := len(pkg), len(pkg), 0
+	if leftPackageSize >= maxPacketLen {
+		s.packetLock.Lock()
```

Comment:
only  leftPackageSize > maxPacketLen need lock, equal not needed?

Reply:
address